### DISCUSSION TITLE
Rustdesk server CLI

### DISCUSSION THREAD

**Discussion by saidlisad** [rustdesk/rustdesk-server-pro#442] [General]
Can i manage serer via CLI on my host?
I want to disable local user and use only azure ad users for logging in.
But for situations, when oidc might break, I would be able to enable the local user via CLI

**Comment by rustdesk** [comment]
You can only use the API, https://github.com/rustdesk/rustdesk/wiki/FAQ#api-of-rustdesk-server-pro
